manifests/BellSoft/LibericaJDK/8.0.252.9.yaml

```diff
@@ -0,0 +1,14 @@
+Id: BellSoft.LibericaJDK
+Name: Liberica JDK 8
+Version: 8.0.252.9
+Publisher: BellSoft
+Homepage: https://bell-sw.com
+License: GPLv2 with exceptions
```

Comment:
This should be `GPL 2 with Classpath Exception`.

Reply:
Thank you. Fixed.